portafly/src/pages/Login.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from 'react'
+import { Redirect } from 'react-router-dom'
+import { useAuth } from 'auth'
+import { useTranslation } from 'i18n/useTranslation'
+import { useDocumentTitle } from 'components/util'
+import {
+  ActionGroup,
+  Button,
+  Form,
+  FormGroup,
+  TextInput,
+  LoginPage,
+  BackgroundImageSrc
+} from '@patternfly/react-core'
+import brandImg from 'assets/logo.svg'
+import BackgroundImageSrcLg from 'assets/images/pfbg_1200.jpg'
+import BackgroundImageSrcSm from 'assets/images/pfbg_768.jpg'
+import BackgroundImageSrcSm2x from 'assets/images/pfbg_768@2x.jpg'
+import BackgroundImageSrcXs from 'assets/images/pfbg_576.jpg'
+import BackgroundImageSrcXs2x from 'assets/images/pfbg_576@2x.jpg'
+
+interface ILoginForm {
+  token: string
+  onChange: (value: string) => void
+  onClick: () => void
+}
+
+const backgroundImages = {
+  [BackgroundImageSrc.lg]: BackgroundImageSrcLg,
+  [BackgroundImageSrc.sm]: BackgroundImageSrcSm,
+  [BackgroundImageSrc.sm2x]: BackgroundImageSrcSm2x,
+  [BackgroundImageSrc.xs]: BackgroundImageSrcXs,
+  [BackgroundImageSrc.xs2x]: BackgroundImageSrcXs2x
+}
+
+const LoginForm: React.FunctionComponent<ILoginForm> = (
```

Comment:
Maybe extract `LoginForm` to it's own file, instead of having 2 components in the same file?

Reply:
I don't think it's worthy, really. This, as the description says, is a temporary solution. We might get rid of the entire file, better to have everything in the same place

Reply:
OK, I get it